src/nspawn/nspawn-register.c

```diff
@@ -47,6 +48,16 @@ static int append_machine_properties(
                                   "char-pts", "rw");
         if (r < 0)
                 return bus_log_create_error(r);
+        if (enable_fuse) {
+                r = sd_bus_message_append(m, "(sv)", "DeviceAllow", "a(ss)", 1,
+                                          /* Allow the container to
+                                           * use FUSE, if
+                                           * copy_devnodes() included
+                                           * it. */
```

Comment:
Too many line breaks...

Reply:
I agree, but it's matching the style of the existing `/dev/net/tun` and `char-pts` comments just a few lines up.

Reply:
Yeah, I know, but please do not wrap like that. If you want, please rebreak the comments in the above.

Reply:
But also, I realize that it's not a very good comment anymore because `copy_devnodes()` isn't the decider since v2.

Reply:
I've added a commit re-flowing the existing comments. But also I've just dropped this comment.